### ISSUE TITLE
Quck Settings Tile for Orbot

### ISSUE THREAD

**Issue by tristan-k** [n8fr8/orbot#181]
With Afwall being able to route apps through Orbot I would like to see [support](https://medium.com/androiddevelopers/quick-settings-tiles-e3c22daf93a8) for Quick Settings Tile in order to quickly toggle Orbot.

**Comment by T0M0F** [comment]
Is there any progress regarding this issue? Otherwise is this project open for contributions? I'd like to try to implement this feature.

**Comment by n8fr8** [comment]
Yes, for sure, please take a shot at it. We will review and merge if it is written decently enough and works.

**Comment by meenbeese** [comment]
Since my PR was merged, this issue can be closed @n8fr8 (sorry for the ping btw)
